tests/modules/vbn_create_stimulus_table/test_create_stim_table.py

```diff
@@ -201,8 +231,12 @@ def mock_get_falling_edges(stim_line, units, **kwargs):
             [1, 1, 1, 1, 1],
             # tolerance
             0,
-            # expected frame offsets
+            # expected expected behavior
```

Comment:
why does the word "expected" occur twice here (and below)?

Reply:
The double expected is a typo. Thanks for finding this.

Reply:
This is fixed. See https://github.com/AllenInstitute/ecephys_etl_pipelines/commit/717cc72c832ffa962c529d1f9bc78c0f10db8649